.github/workflows/tests.yml

```diff
@@ -0,0 +1,60 @@
+name: tests
+
+on:
+  push: null
+  pull_request: null
+
+jobs:
+  tests:
+    name: tests
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: cancel previous runs
+        uses: styfle/cancel-workflow-action@0.6.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: 3.8
+          channels: conda-forge,defaults
+          channel-priority: strict
+          show-channel-urls: true
+
+      - name: configure conda and install code
+        shell: bash -l {0}
+        run: |
+          conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
+          conda update --quiet conda
+          conda config --add channels conda-forge --force
```

Comment:
Do we need this twice? Or the setup-miniconda options don't work?

Reply:
I'm mostly trying to produce a hybrid between our file at `cf-scripts` and the existing CI script. Our at `cf-scripts` does have a bunch of conda config wrangling.